BitwardenShared/Core/Vault/Models/API/CipherSSHKeyModel.swift

```diff
@@ -0,0 +1,14 @@
+/// API model for an SSH key.
+///
+struct CipherSSHKeyModel: Codable, Equatable {
+    // MARK: Properties
+
+    /// The public key of the SSH key.
+    let publicKey: String?
+
+    /// The private key of the SSH key.
+    let privateKey: String?
+
+    /// The key fingerprint of the SSH key.
+    let keyFingerprint: String?
```

Comment:
🎨 We've been alphabetizing model properties elsewhere, does it make sense to do the same here?

Reply:
Nice catch, yep makes perfect sense; thanks Matt!